week4/MealsharingNet/Models/Review.cs

```diff
@@ -0,0 +1,10 @@
+namespace MealsharingNet.models;
```

Comment:
The namespace here should be with uppercase, namespace MealsharingNet.**Models**;